ml-data-hub-plugin/src/test/groovy/com/marklogic/gradle/task/UpdateIndexesTaskTest.groovy

```diff
@@ -92,8 +92,8 @@ class UpdateIndexesTaskTest extends BaseTest {
 		result.task(":mlUpdateIndexes").outcome == SUCCESS
 
         //Test to verify range-path-index for mlUpdateIndex
-		assert (getStagingIndexValuesSize('//m:range-path-index') == 2)
-		assert (getFinalIndexValuesSize('//m:range-path-index') == 2)
+		assert (getStagingIndexValuesSize('//m:range-path-index') == 3)
```

Comment:
I think the better test to enhance is DeployDatabaseFieldCommandTest. See my comments below about what to do with UpdateIndexesTaskTest, which I don't think is proving much of anything.